analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Binary.cs

```diff
@@ -620,8 +609,7 @@ public void Binary_NullableRelationalNonNull_SetsObjectConstraint_CS(string expr
             """;
         var validator = SETestContext.CreateCS(code, "int? arg").Validator;
         validator.ValidateContainsOperation(OperationKind.Binary);
-        validator.ValidateTag("If", x => x.HasConstraint(ObjectConstraint.NotNull).Should().BeTrue("arg comparison true, hence non-null"));
-        validator.ValidateTag("If", x => x.HasConstraint<NumberConstraint>().Should().BeTrue("arg inferred a value from the comparison"));
+        validator.TagValue("If").Should().HaveOnlyConstraints(new SymbolicConstraint[] { ObjectConstraint.NotNull, NumberConstraint.From(min, max) }, "arg comparison true, hence non-null");
```

Comment:
There were 2 assertions merged, and only one comment preserved. I don't think it makes sense now when we have this precise assertion in place.
```suggestion
        validator.TagValue("If").Should().HaveOnlyConstraints(new SymbolicConstraint[] { ObjectConstraint.NotNull, NumberConstraint.From(min, max) });
```